### PR TITLE
Trim client.prep

### PR DESCRIPTION
**Pull request by spuiuk**
We had created 
a) Mount points in /test-mnt/ and entries in /etc/hosts
b) fuse mountpoint in /test-mnt/fuse
with the intention to make testing easier. 

Since then, we have introduced a test runner which runs various tests.  It is now recommended to do all mounting using the helpers from the test runner. These mount points are no longer needed.

### CODE REVIEW

File: vagrant/ansible/roles/client.prep/tasks/main.yml [hunk 1]
```diff
@@ -15,49 +15,6 @@
 - debug:
     msg: "{{ ctdb_network_public_interfaces }}"
```
Comment (anoopcs9) [inline vagrant/ansible/roles/client.prep/tasks/main.yml:16]: Should we remove this debug section too?

Comment (spuiuk) [reply vagrant/ansible/roles/client.prep/tasks/main.yml:16]: Probably should but in a different PR I guess. That particular change isn't related to the other fixes which remove the mount points and any changes to this commit message at this stage means waiting for the build fixes to go in.

Comment (anoopcs9) [reply vagrant/ansible/roles/client.prep/tasks/main.yml:16]: Fine.

**Comment by anoopcs9** [comment]
retest this please

**Comment by anoopcs9** [comment]
Waiting for additional merge commit to be removed before merging...

**Pull request merged**